internal/config/template.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import(
+	"html/template"
+)
+
+var (
+	Tmpl *template.Template
+	err error
+)
+
+func GetTemplates() error{
+	Tmpl, err = template.ParseGlob("templates/*")
+	if err != nil{
+		return err
+	}
+	return nil
+}
```

Comment:
@kwsorensen I went back and forth about putting this in it's own package, but ultimately decided it kinda fits in the `config` package. wdyt?

Reply:
Either we should make an entire "frontend config" package or just throw it in main.go

Reply:
I think it needs to be in a package (not just in `main.go`) so it can be referenced from other packages (like the notes package) but wasn't sure if this small bit of code needed it's own package or if it loosely fell under "config" 🤷 

If you think a `frontend` package is better, I can definitely create that

Reply:
decided on keeping this in the `config` package for now